client/ayon_core/hosts/nuke/plugins/publish/collect_headless_farm.py

```diff
@@ -0,0 +1,42 @@
+import pyblish.api
+
+
+class CollectHeadlessFarm(pyblish.api.ContextPlugin):
+    """Setup instances for headless farm submission."""
+
+    order = pyblish.api.CollectorOrder + 0.4999
```

Comment:
Why this happens so late?

Reply:
To be the last plugin to run.

Reply:
But it can be in fact one of the first plugins instead. It can run after `CollectFromCreateContext` which has order `pyblish.api.CollectorOrder - 0.5`, or?

Reply:
We need to make sure `instance.data["families"] = ["headless_farm"]` only. Cause we do not want any `review` or other families.

Reply:
But all the instances you're disabling here will be collecte and will go through all collectors before you actually disable them. So this can be moved to `pyblish.api.CollectorOrder - 0.49`.

Reply:
And to be honest I don't think it is good idea to clear `"families"` this late, the instance won't be integrated to ftrack.

Reply:
> But all the instances you're disabling here will go through all collectors before you actually disable them, my question is why to not disable them as soon as possible? This can be moved to pyblish.api.CollectorOrder - 0.49.

Have split the plugin to disable instances as early as possible now.

> And if that is intended behavior then I suggest to move it out of this plugin to another one which can be enabled/disabled, so this logic can be in fact used as "standard" publishing.

I dont get what you mean here?

Reply:
> I dont get what you mean here?

If clearing of `"families"` is for some reason intended behavior, then it probably should be in specific plugin that does that (later in the order).